Controller/Payment/Order.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Razorpay\Magento\Controller\Payment;
 
+use Razorpay\Api\Api;
```

Comment:
Why this referenced?

Reply:
To call preference API,
```
$api = new Api($this->config->getKeyId(),"");
$response = $api->request->request("GET", "preferences");
```

Reply:
But Order create is already using API reference, before as well.